NDB.Covid19/NDB.Covid19.iOS/Views/ErrorStatus/ErrorPageViewController.cs

```diff
@@ -19,11 +21,38 @@ public override void ViewDidLoad()
 		{
 			base.ViewDidLoad();
 			SetupStyling();
-			
+			AddObservers();
+		}
+
+		public override void ViewWillAppear(bool animated)
+		{
+			base.ViewWillAppear(animated);
+            Covid19.Utils.LogUtils.LogMessage(Enums.LogSeverity.INFO, "The user is seeing General Error", null, GetCorrelationId());
```

Comment:
Indent is weird here

Reply:
Not sure what happened here. It seemed fine in visual studio.
I removed indentation and added it aging, and it seems to have resolved the issue.